examples/storage_demo.yaml

```diff
@@ -110,7 +107,7 @@ file_mounts:
   # other storage mounts using the same bucket with MOUNT mode. Note that the
   # source is synced with the remote bucket everytime this task is run.
   /dataset-storage-mount:
-    name: sky-dataset-romilzz
+    name: sky-dataset-mybucket
```

Comment:
For this line and L94, will we upload the `~/datasets` only once? As a future TODO, we may want to do a unique for the `(source, name)` pair.

Reply:
`gsutil rsync`/`aws s3 sync` will be invoked twice, but the contents will be uploaded only once. I think the motivation to use `~/datasets` at L110, L94 and L71 is to show that the same directory can be made available at the remote VM in different ways